wal.go

```diff
@@ -125,17 +160,21 @@ func getKeyValueEntryFromOffsetViaData(offset int64, data []byte) (*KeyValueEntr
 	// get value
 	v := data[offset+ValueSizeOffset+intKsz : offset+ValueSizeOffset+intKsz+intVsz]
 
-	// make keyValueEntry
-	x := &KeyValueEntry{
-		deleted: deleted,
-		tstamp:  int64(tstamp64Bit),
-		ksz:     int64(intKsz),
-		vsz:     int64(intVsz),
-		k:       k,
-		v:       v,
-		offset:  offset,
-		size:    StaticChunkSize + intKsz + intVsz,
+	x := NewKeyValueEntry(
```

Comment:
better naming for this variable. Maybe `keyValueEntryFromOffset` or `newKeyValueEntry`?